hack/jenkins/e2e.sh

```diff
@@ -434,6 +434,18 @@ case ${JOB_NAME} in
     : ${PROJECT:="kubernetes-jenkins"}
     ;;
 
+  # Runs slow tests on GCE 1.1 cluster, sequentially.
+  kubernetes-gce-slow-release-1.1)
```

Comment:
You need to actually configure it to use the `release-1.1` branch:

```
${JENKINS_PUBLISHED_VERSION:="ci/latest-1.1"}
```

See the other 1.1 jobs.